src/app/permission/app-search/app-search.component.spec.ts

```diff
@@ -14,10 +13,14 @@ import {
   WorkspaceAbstract,
   WorkspacePageResult,
   ApplicationPageResult,
-  Application
+  Application,
+  AssignmentAPIService,
+  Permission
 } from 'src/app/shared/generated'
 import { App, AppSearchComponent } from './app-search.component'
 import { RowListGridData } from '@onecx/angular-accelerator'
+// import { FileSelectEvent } from 'primeng/fileupload'
```

Comment:
do we need this import?

Reply:
I wasn't able to get the related test to run and commented it out. That's why this is also commented out